src/main/views/terms-and-conditions.njk

```diff
@@ -0,0 +1,187 @@
+        Entering sensitive information is done so at your own risk.') }}
+      </p>
+
+
+      <h2 class="heading-medium">{{ t('Information about you and your visits to GOV.UK') }}</h2>
+
+      <p>
+        {{ t('We collect information about you in accordance with our privacy policy and our cookie policy') }}
+      </p>
+
+
+
+      <h2 class="heading-medium">{{ t('Disclaimer') }}</h2>
+
+      <p>
+        {{ t('While we make every effort to keep GOV.UK up to date, we don’t provide any guarantees,
+        conditions or warranties that the information will be:') }}
+      </p>
+
+      <ul class="list list-bullet">
+        <li>{{ t('current') }}</li>
+        <li>{{ t('secure') }}</li>
+        <li>{{ t('accurate') }}</li>
+        <li>{{ t('complete') }}</li>
+        <li>{{ t('free from bugs or viruses') }}</li>
+      </ul>
+
+      <p>
+        {{ t('We don’t publish advice on GOV.UK. You should get professional or specialist advice before doing anything
+         on the basis of the content.') }}
+      </p>
+
+      <p>
+        {{ t('We’re not liable for any loss or damage that may come from using GOV.UK. This includes:') }}
+      </p>
+
+      <ul class="list list-bullet">
+        <li>{{ t('any direct, indirect or consequential losses') }}</li>
+        <li>{{ t('any loss or damage caused by civil wrongs
+        (‘tort’, including negligence), breach of contract or otherwise') }}</li>
+        <li>{{ t('the use of GOV.UK and any websites that are linked to or from it') }}</li>
+        <li>{{ t('the inability to use GOV.UK and any websites that are linked to or from it') }}</li>
+      </ul>
+
+      <p>
+        {{ t('This applies if the loss or damage was foreseeable, arose in the normal course of things or you advised us that it might happen.') }}
+      </p>
+
+      <p>
+        {{ t('This includes (but isn’t limited to) the loss of your:') }}
+      </p>
+
+      <ul class="list list-bullet">
+        <li>{{ t('income or revenue') }}</li>
+        <li>{{ t('salary, benefits or other payments') }}</li>
+        <li>{{ t('business') }}</li>
+        <li>{{ t('profits or contracts') }}</li>
+        <li>{{ t('opportunity') }}</li>
+        <li>{{ t('anticipated savings') }}</li>
+        <li>{{ t('data') }}</li>
+        <li>{{ t('goodwill or reputation') }}</li>
+        <li>{{ t('intangible property, including loss,
+        corruption or damage to data or any computer system') }}</li>
+        <li>{{ t('wasted management or office time') }}</li>
+      </ul>
+        <p>
+          {{ t('We may still be liable for:') }}
+        </p>
+
+
+        <ul class="list list-bullet">
+          <li>{{ t('death or personal injury arising from our negligence') }}</li>
+          <li>{{ t('fraudulent misrepresentation') }}</li>
+          <li>{{ t('any other liability which cannot be excluded or limited under applicable law') }}</li>
+        </ul>
+
```

Comment:
lots of extra spacing in these files can you remove some of the extra blank lines please

Reply:
Ive separated each block of code by 1 line to easily separate it for readability reasons.
However, I've removed all double spacing.